content/projects/gsoc/2019/application.md

```diff
@@ -0,0 +1,120 @@
+
+* Each student project will have at least 2+ mentors AND org admin advisor assigned to the project.
+* Mentors are expected to be accomplished Jenkins contributors, who are passionate about the community/mentorship work.
+* Mentors bring their own project, and they are expected to have a high interest in it
+* Mentors will be directly involved into student selection and interview processes so they will establish connection with successful students early
+* We do not require mentors to be experts in the code base, because we expect students to work with other community members.
+* We will conduct Weekly meetings between mentors and org admins to sync-up on progress and any issues.
+* As org admins we will be monitoring mentor/student interaction starting from the application phase. If something goes wrong, as org admins we will jump in and/or find additional mentors
+* There will be a private communication channel between org admins and mentors.
+
+#### How will you help your students stay on schedule to complete their projects?
+
+* During the Community Bonding phase mentors will share their expertise in order to define realistic project plans, design document, and effort estimations.
+* The student project milestones will be discussed and confirmed between mentors and students. Milestones will be aligned with student evaluations and will have clear expectations set up.
+* Mentors will set up regular meetings with students (at least twice per week) in order to sync-up on projects. We will also recommend having retrospectives with students after evaluations.
+* Mentors should be available to questions. They should also provide a periodic feedback on the progress of the project and on the performance of particular students (1x1).
+* We will set up weekly public office-hours with students (or two meetings if time-zones require it) and private ones between mentors and org admins to sync-up
+* We will be requiring a daily push to branches so that the students always show the progress and keep changes atomic.
+* During the coding period, mentors will participate in the code reviews of their student and report to org admins if the project stalls.
+
+#### How will you get your students involved in your community during GSoC?          
+* Students will cooperate with Jenkins community during the project. Org Admins will provide an introductory training (community overview, code-of-conduct, etc.), then mentors will help students to establish contacts with experts from the community
+* We will ensure that students are around in public chats and other communication channels during the “working days”
+* Students will be encouraged to present proofs-of-concept of their projects on public Jenkins Developer meetings and in blog posts
+* Students will be involved into all standard processes in our community (including pull requests, code reviews, IRC/Gitter discussions, test automation for their projects, documentation development, etc.).
```

Comment:
Do we have standards for what constitutes adequate test coverage?

Reply:
Also when - do we expect unit tests to be included for each PR?

Reply:
Nope. "Please have some tests" would be a good advice in https://jenkins.io/projects/gsoc/students/ , but I would rather prefer to leave the detailed requirements to mentors